src/us_reverse_geo/Result.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * A candidate is a possible match for an address that was submitted.<br>
+ *     A lookup can have multiple candidates if the address was ambiguous.
+ *
+ * @see "https://smartystreets.com/docs/cloud/us-reverse-geo-api#result"
+ */
+class Result {
+	constructor(responseData) {
+		this.coordinate = {};
+		if (respoonseData.coordinate !== undefined) {
```

Comment:
typo fix and you should be able to simplify because it should come out truthy if it's not `undefined` (unless it was set to `false`, which means it's not `undefined`, but that shouldn't apply in this case)
```suggestion
		if (responseData.coordinate) {
```